pkg/project/cloud/Taskfile.yml

```diff
@@ -149,6 +150,25 @@ tasks:
     cmds:
       - sh aemw package deploy --file "all/target/*.all-*.zip"
 
+  aem:check:
```

Comment:
Shouldn't it be internal?

Reply:
I believe not; after fixing sth on AEM and rechecking you may not want to call `setup` again